diffgraph/processing_modes/tree_sitter_dependency.py

```
@@ -0,0 +1,1185 @@
+                                    
+                                    if name_node:
+                                        method_name = source_bytes[name_node.start_byte:name_node.end_byte].decode('utf-8')
+                                        components.append(ExtractedComponent(
+                                            name=method_name,
+                                            component_type="method",
+                                            parent=class_name,
+                                            start_line=member.start_point[0],
+                                            end_line=member.end_point[0],
+                                            file_path=file_path
+                                        ))
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)